gvm/protocols/gmpv7/gmpv7.py

```diff
@@ -2777,7 +2777,7 @@ def get_alerts(
 
         return self._send_xml_command(cmd)
 
-    def get_alert(self, alert_id: str) -> Any:
+    def get_alert(self, alert_id: str, tasks: Optional[bool] = None) -> Any:
```

Comment:
Optional arguments should always be required to be passed as keywords.
```suggestion
    def get_alert(self, alert_id: str, *, tasks: Optional[bool] = None) -> Any:
```